src/app/scheduling/daterange/date-range-form/daterange-form.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="content-wrap">
+    <form class="book-flight-form" [formGroup]="twoInputForm" (ngSubmit)="submitTwoInputForm()">
+        <div class="container">
+            <div class="input-wrapper">
+                <igx-input-group class="contentA">
+                    <input #departureAirport igxInput type="text" formControlName="departureAirport">
+                    <label for="departureAirport" igxLabel>From</label>
+                    <igx-suffix>
+                        <igx-icon>plane</igx-icon>
+                    </igx-suffix>
+                </igx-input-group>
+                <igx-input-group class="contentB">
+                    <input #arrivalAirport igxInput type="text" formControlName="arrivalAirport">
+                    <label for="arrivalAirport" igxLabel>To</label>
+                    <igx-suffix>
+                        <igx-icon>plane</igx-icon>
+                    </igx-suffix>
+                </igx-input-group>
+                <div>
+                </div>
+                <!-- <div class="date-range-wrapper"> -->
+                    <igx-date-range class="date-range-wrapper" [weekStart]="1" [todayButtonText]="'Show Today'" [monthsViewNumber]="1"
```

Comment:
This tag is wrong, it should be `igx-date-range-picker`. Also the range picker does not have a `todayButtonText` or a `monthsViewNumber`.